packages/support/resources/views/components/section/index.blade.php

```diff
@@ -66,7 +66,7 @@
                 } => ! $aside,
             ])
         >
-            <div class="flex items-center gap-3 overflow-hidden">
+            <div class="flex items-center gap-3 overflow-hidden px-px">
```

Comment:
Did you guys mean `p-px`?
I don't think that's a nice fix though.
Have you checked git blame to see why overflow-hidden is here in the first place, @bennettblack?

Reply:
@zepfietje [Commit Here](https://github.com/filamentphp/filament/commit/36a1aa72abe7878293ad4dece334b4b79dab9f37) Seems that it addresses a previous UI issue.

Reply:
overflow-hidden was in the markup before that commit

Reply:
Yeah, this was only a recent rework by Dan.

Reply:
I've used git blame to go back to the creation of the section component, and I can't find the reason `overflow-hidden` is there. I'm just going to remove it, as it causes other issues and I don't currently see why it's needed.